src/js/components/RadioButtonGroup/doc.js

```diff
@@ -29,8 +29,8 @@ export const doc = RadioButtonGroup => {
       elements.`,
     ).isRequired,
     onChange: PropTypes.func.description(
-      `Function that will be called when the user clicks on of the radio
-      buttons. It will be passed a React event object.`,
+      `Function that will be called when the user clicks on one of the radio
+      buttons. It will be passed as a React event object.`,
```

Comment:
The "one" change is right. The "as" change means "The function will be passed as a React event object.", which I don't think is right. I think the original wording for this sentence is better.